.circleci/config.yml

```diff
@@ -1421,6 +1421,43 @@ jobs:
       - store_artifacts:
           path: webui/react/coverage
 
+  test-unit-react-screenshot:
+    docker:
+      - image: cimg/node:16.19.0
```

Comment:
>we're using undici for our node fetch needs because it's the source of the [experimental fetch library that landed in node 18](https://nodejs.org/en/blog/announcements/v18-release-announce#fetch-experimental), so when we upgrade, we should be able to just remove the import and library and everything will be the same.

Is this test running only on CI? If so, this can be node 18 because this job doesn't rely on anything.

Reply:
I think it'd be for the best if we kept our node versions consistent across CI.